crates/turbopack-core/src/issue/mod.rs

```diff
@@ -73,6 +73,41 @@ impl Display for IssueSeverity {
     }
 }
 
+#[derive(Clone, Debug)]
+#[turbo_tasks::value(shared)]
+pub enum StyledString {
```

Comment:
not going to suggest to make any changes, just random thoughts that if we could just pass raw markdown or formatted text and consumer (next-swc) can transparently renders it'd be cool